.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="SquareAndroid" />
```

Comment:
do not use the SquareAndroid style in this project, we have a customised default project setting